src/defaults/html.pipe.js

```diff
@@ -44,6 +44,10 @@ function hascontent({ content }) {
   return !(content !== undefined && content.body !== undefined);
 }
 
+function paranoid(context, action) {
+  return action && action.secrets && !!action.secrets.HLX_SANITIZE_DOM;
```

Comment:
@trieloff shouldn't `HLX_SANITIZE_DOM` also be added to https://github.com/adobe/helix-pipeline/blob/master/src/schemas/secrets.schema.json?